build/nightly-mini-tester.sh

```diff
@@ -65,12 +65,6 @@ find ${prefix}/Calpont -type f | xargs chmod +r
 
 mkdir -p ${prefix}/Calpont/data1/systemFiles/dbrm
```

Comment:
While not related to the pull request - i see references to Calpont directories, i wonder if this is even exercised since the installs should be in columnstore now?  Similarly in src-build below.